tests/python/openassetio_mediacreation/trait/test_content.py

```diff
@@ -108,4 +102,4 @@ def an_empty_traitsData():
 
 @pytest.fixture
 def a_locatableContent_traitsData():
-    return TraitsData({"locatableContent"})
+    return TraitsData({"openassetio-mediacreation:content.LocatableContent"})
```

Comment:
[Question] Is this sort of project namespaced trait name what folks were expecting?

Reply:
Yeah - that's spot on.

Reply:
It makes sure that you never get conflicts from people accidentally using the same trait name in two packages